enumeration/remote/aws/s3_bucket_analytic_enumerator.go

```diff
@@ -69,7 +70,7 @@ func (e *S3BucketAnalyticEnumerator) Enumerate() ([]*resource.Resource, error) {
 					string(e.SupportedType()),
 					id,
 					map[string]interface{}{
-						"region": region,
+						"alias": region,
```

Comment:
no sure I understand this change ˆˆ

Reply:
If you read the previous func that defined read attributes for those s3 related resources, we were retrieviing the regions attribute and setting it to the "alias" field. Since we do not have this method anymore, I needed to change it there directly.

![image](https://user-images.githubusercontent.com/6154987/182110674-f1952fba-d43d-4770-858e-65bfcce197c3.png)

Does it answer your question ?